installer/conf/omsagent.d/patch_management.conf

```diff
@@ -23,4 +23,29 @@
 <filter oms.patch_management_immediate_run>
   type filter_patch_management
   log_level warn
+</filter>
+
+<source>
+    @type tail
+    tag oms.update_progress
+    path /var/log/apt/history.log
+    pos_file /var/opt/microsoft/omsagent/run/history.log.pos
+    keep_time_key true
+    format multiline
+    format_firstline /Start-Date:/
+    format1 /Start-Date: (?<start-date>.*)\n/
+    format2 /Commandline: (?<apt-logs>.*\n)/
+    read_from_head true
+    log_level trace
```

Comment:
Does the history.log get recycled? On one of my Ubuntu machines where I installed java though apt-get under root user, the history.log for some reason is empty. Was the empty history.log scenario tested?